### PR TITLE
Add custom redirect warning

### PR DESCRIPTION
**Pull request by gnufied**
Add a warning for redirecting to external websites.

xref https://github.com/kubernetes-csi/docs/issues/495

Screenshot from 2022-02-23 15-35-18

cc @msau42 @saad-ali @xing-yang

### CODE REVIEW

File: book/custom.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+document.querySelectorAll(".production_drivers a").forEach(function(externalLink){
+  externalLink.addEventListener("click", function(e){
+    if (confirm("You are being redirected to an external website.\nKubernetes community does not validate external CSI drivers and they should be used at your own risk.") == true) {
```
Comment (msau42) [inline book/custom.js:3]: Tiny nit: "The Kubernetes community"

Comment (gnufied) [reply book/custom.js]: fixed.

**Pull request merged**